slicer.py

```diff
@@ -131,9 +131,11 @@ def parse_currency_data(filename):
                 if localization.lower() in fields[0].lower():
                     currency = 'USD - RoW'
  
-        exchange_rate = Decimal(fields[column_index_exchange_rate].replace(',', ''))
         amount_pre_tax = Decimal(fields[column_index_amount_pre_tax].replace(',', ''))
         amount_after_tax = Decimal(fields[column_index_amount_after_tax].replace(',', ''))
+        earnings = Decimal(fields[column_index_earnings].replace(',', ''))
+        exchange_rate = earnings/amount_pre_tax        
```

Comment:
It is necessary to use `amount_after_tax` to get the correct exchange rate for cases where there is a withholding tax.
```suggestion
        exchange_rate = earnings / amount_after_tax        
```

Reply:
That's correct. Thank you!

Reply:
Sorry, you need to actually implement the suggested fix. Right now, you’ve just marked this conversation as resolved.

Reply:
Done